radicle-cli/src/commands/review.rs

```diff
@@ -118,7 +119,7 @@ impl Args for Options {
 
         Ok((
             Options {
-                id: id.ok_or_else(|| anyhow!("a patch id to review must be provided"))?,
+                id: id.ok_or_else(|| anyhow!("a patch-id to be reviewed, must be provided"))?,
```

Comment:
"patch to review" is correct and less verbose

Reply:
https://github.com/radicle-dev/heartwood/pull/445/commits/6d0b338c6920de263d264d7bac5e5c819bc80511#diff-0a81e1fb555fea14804c21b172dad499b49d4348f97e96bdce1638b97e90a27dR122